src/language-js/print/array.js

```diff
@@ -171,6 +172,36 @@ function isConciselyPrintedArray(node, options) {
 }
 
 function printArrayElements(path, options, elementsProperty, print) {
+  const elements = path.node[elementsProperty];
+
+  function isLineAfterElementEmpty(elementNode) {
+    const text = options.originalText;
+    const elementIdx = elements.findIndex((v) => v == elementNode);
+    const nextNode = elements.find((e, i) => i > elementIdx && e !== null);
```

Comment:
path.siblings, path.next, path.index should be used

Reply:
oh, you're right, that does make sense, making these changes now